.github/workflows/publish-snapshot-docker.yml

```diff
@@ -29,14 +29,22 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
       - name: Login to Docker Hub
         uses: docker/login-action@v2
         with:
           username: ${{ secrets.DOCKERHUB_USER }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
-      - name: Build Kyuubi Docker Image
-        run: docker build --tag apache/kyuubi:master-snapshot --file build/Dockerfile .
-      - name: Docker image
-        run: docker images
-      - name: Push Docker image
-        run: docker push apache/kyuubi:master-snapshot
+      - name: Build and Push Kyuubi Docker Image
+        uses: docker/build-push-action@v4
+        with:
+          context: .
+          platforms: linux/amd64,linux/arm64
+          push: true
+          file: build/Dockerfile
+          cache-from: type=gha
```

Comment:
What does `gha` stands for? Didn't find a clue on https://github.com/docker/build-push-action .

Reply:
It is cache on Github Actions
Docs: https://docs.docker.com/build/cache/backends/gha/#using-dockerbuild-push-action

Reply:
Thanks for the hint. Better to attach this information to the description for future reference.

Reply:
Got it. thanks for suggestion.